tests/cypress/integration/actions_projects_models/case_104_project_export_3d.js

```diff
@@ -21,7 +21,7 @@ context('Export project dataset with 3D task.', { browser: '!firefox' }, () => {
     };
     let projectID = '';
 
-    function getProjectID(projectName) {
+    function getProjectID() {
```

Comment:
In line 35 we again pass argument projectName to this function (maybe there are other places where it is called, I didn't check)

Reply:
Fixed.